src/libs/Navigation/AppNavigator/ReportScreenIDSetter.ts

```diff
@@ -39,6 +41,13 @@ function ReportScreenIDSetter({route, reports, policies, navigation, isFirstTime
     const {canUseDefaultRooms} = usePermissions();
 
     useEffect(() => {
+        const reportActionID = route?.params?.reportActionID;
+        const regexValidReportActionID = new RegExp(/^\d*$/);
+        if (reportActionID && !regexValidReportActionID.test(reportActionID)) {
+            Navigation.goBack(ROUTES.HOME);
```

Comment:
@roryabraham I am thinking we should go to report without any reportId in this case. What do you think?

Reply:
No, I think HOME is ok. It's the same effect as report w/o any reportID, right?